memory/title_sequence_manager.py

```diff
@@ -39,12 +41,14 @@ def update(self):
                 # Update fields
                 self.title_position_set = False
                 self.get_load_save_done()
+                self.get_pressed_start()
```

Comment:
As a general comment on these; the methods may be somewhat poorly named. Were I to just take a glance at this code, I'd assume that `TitleSequenceManager::get_pressed_start()` is a method that returns if start is pressed. Instead, I have to use the property on the object, `start_pressed`.

This could be somewhat alleviated by "marking the method as private". Python doesn't really have this feature, but by convention starting a method name or property with `_` indicates that nothing outside the class should be messing with it.

Could also change the `get_x()` to `_load_x()` or something similar to further clarify.